client/i18n/locales/english/translations.json

```diff
@@ -365,6 +365,7 @@
     "question": "Question",
     "solution-link": "Solution Link",
     "github-link": "GitHub Link",
```

Comment:
```suggestion
```
Replaced by source-code-link

Reply:
I was hesitant to remove it, because it seems useful for future, and is already translated.